test/infrastructure/net-operator/controllers/networkinterface_controller.go

```diff
@@ -0,0 +1,123 @@
+/*
+Copyright 2024 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package controllers
+
+import (
+	"context"
+	"time"
+
+	"github.com/pkg/errors"
+	corev1 "k8s.io/api/core/v1"
+	apierrors "k8s.io/apimachinery/pkg/api/errors"
+	kerrors "k8s.io/apimachinery/pkg/util/errors"
+	"sigs.k8s.io/cluster-api/util/patch"
+	"sigs.k8s.io/cluster-api/util/predicates"
+	ctrl "sigs.k8s.io/controller-runtime"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+	"sigs.k8s.io/controller-runtime/pkg/controller"
+	"sigs.k8s.io/controller-runtime/pkg/reconcile"
+
+	"sigs.k8s.io/cluster-api-provider-vsphere/test/framework/vmoperator"
+	netopv1alpha1 "sigs.k8s.io/cluster-api-provider-vsphere/test/infrastructure/net-operator/external/net-operator/api/v1alpha1"
```

Comment:
We already import `github.com/vmware-tanzu/net-operator-api/api/v1alpha1`, is there a reason why we want to copy this time from there?

Reply:
The first time I tried this I got some go mod update issue, and I just got around it quicky and moved on
But probably I was doing something wrong and now it worked!
Thank you for reminding me about this!